README.md

```diff
@@ -1,11 +1,40 @@
-This repository contains Theia extensions related to C/C++ development.
+<div align='center'>
+<br />
+<img src='./logo/theia.svg' alt='theia logo' width='125'>
 
-The `browser-app` and `electron-app` directories contain examples of Theia-based applications containing these extensions.
+<h2>ECLIPSE THEIA - C/C++ EXTENSION</h2>
```

Comment:
extensionS ?

Reply:
I put `EXTENSION` since it is the name of the repository, and it's a C/C++ extension with multiple functionalities. 
Do you think `EXTENSIONS` would be more appropriate?

Reply:
Maybe we can rename the repo later, better sooner?

It was originally created to host the `@theia/cpp` extension, but now it is already a monorepo containing 2 extensions, 3 once `@theia/cpp` is moved over...

So I would write "EXTENSION**S**", since it hosts multiple extensions.

Reply:
I can update it :)

I think a better name than `extensions` could be used as it's confusing.
The name of the repo would be `extensions` yet it's a single `extension` (dependency) to Theia.

Perhaps something like `@theia/cpp-tools` would be more appropriate or if/when the original `@theia/cpp` extensions is copied over we can simply call the monorepo `@theia/cpp`.